src/_P088_HeatpumpIR.ino

```diff
@@ -45,9 +45,7 @@
 
 #include <HeatpumpIRFactory.h>
 
-HeatpumpIR *heatpumpIR;
-IRSenderIRremoteESP8266 *Plugin_088_irSender;
-
+IRSenderIRremoteESP8266 *Plugin_088_irSender = NULL;
```

Comment:
Just as a note (you don't need to change it).
It is better to use `nullptr` for pointer initialization as the compiler then can treat it differently.  (e.g. compiler checks)